lib/src/configuration.dart

```diff
@@ -36,14 +36,23 @@ class Configuration {
   RealmSchema get schema => _schema;
 
   /// Creates a [Configuration] with schema objects for opening a [Realm].
+  ///
+  /// [fifoFilesFallbackPath] is a custom directory for storing FIFO special files
+  /// in case the Realm file is in a location that does not allow the creation of FIFO special files.
+  ///
   /// [readOnly] controls whether a [Realm] is opened as readonly.
   /// This allows opening it from locked locations such as resources,
   /// bundled with an application.  The realm file must already exists.
-  Configuration(List<SchemaObject> schemaObjects, {bool readOnly = false})
+  Configuration(List<SchemaObject> schemaObjects, {String? fifoFilesFallbackPath, bool readOnly = false})
       : _schema = RealmSchema(schemaObjects),
         _handle = realmCore.createConfig() {
     schemaVersion = 0;
     path = defaultPath;
+
+    if (fifoFilesFallbackPath?.isNotEmpty ?? false) {
```

Comment:
could check for null only instead. if you really want to check for empty string then we need to throw a meaningful exception. I suggest checking for null only here

Reply:
Done.